app/_src/guides/federate-kv.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Federate zone control plane
+---
+
+With {{site.mesh_product_name}} you can first start with just one zone control plane
+and then federate it to a multi-zone deployment.
+This way you can:
+- see your mesh deployment in one centralized place
+- connect multiple zones and introduce cross zone connectivity
+- manage policies that are pushed to all zones
+
+## Prerequisites
+- Completed [quickstart](/docs/{{ page.release }}/quickstart/kubernetes-demo-kv/) to set up a zone control plane with demo application
+- Have [kumactl installed and in your path](/docs/{{ page.release }}/introduction/install/)
+
+{% tip %}
+If you are already familiar with quickstart you can set up required environment by running:
+TODO SLEEP
+```sh
+helm upgrade \
+  --install \
+  --create-namespace \
+  --namespace {{ site.mesh_namespace }} \{% if version == "preview" %}
+  --version {{ page.version }} \{% endif %}
+  {{ site.mesh_helm_install_name }} {{ site.mesh_helm_repo }}
+kubectl apply -f kuma-demo://k8s/001-with-mtls.yaml
+```
+{% endtip %}
+
+## Start a global control plane
+
+### Start a new Kubernetes cluster
+
+Currently, it's not possible to deploy global and zone control plane in the same Kubernetes cluster, therefore we need a new Kubernetes cluster.
+
+{% tip %}
+You can skip this step if you already have a Kubernetes cluster running.
+It can be a cluster running locally or in a public cloud like AWS EKS, GCP GKE, etc.
+{% endtip %}
+
+```sh
+minikube start -p mesh-global
+```
+
+### Setup the minikube tunnel
+If you are using minikube for local testing, you can take advantage of the built-in minikube tunnel command. This command allows load balancer addresses to be provisioned using localhost.
+
+{% tip %}
+Using `nohup` will allow the tunnel to continue running should your current terminal session end. 
+{% endtip %}
+
+```sh
+nohup minikube tunnel -p mesh-global &
+```
+
+### Deploy a global control plane
+
+```sh
+helm install --kube-context=mesh-global --create-namespace --namespace {{site.mesh_namespace}} \
+--set {{site.set_flag_values_prefix}}controlPlane.mode=global \
+--set {{site.set_flag_values_prefix}}controlPlane.defaults.skipMeshCreation=true \
+{{ site.mesh_helm_install_name }} {{ site.mesh_helm_repo }}
+```
+
+We skip default mesh creation as we will bring mesh from zone control plane in the next steps.
+
+### Sync endpoint
+
+Find and save the external IP and port of the `{{site.mesh_cp_zone_sync_name_prefix}}global-zone-sync` service in the `{{site.mesh_namespace}}` namespace:
+
+```shell
+export KDS_IP=$(kubectl --context=mesh-global get svc --namespace {{site.mesh_namespace}} {{site.mesh_cp_zone_sync_name_prefix}}global-zone-sync -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
```

Comment:
Let's be consistent and not use `=` after the flag names whenever it's unnecessary and use `-n` instead of `--namespace` in kubectl
```suggestion
export KDS_IP=$(kubectl --context mesh-global get svc -n {{site.mesh_namespace}} {{site.mesh_cp_zone_sync_name_prefix}}global-zone-sync -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
```